boat_nav/src/sensor_parser_node.py

```diff
@@ -105,7 +106,7 @@ def to_lps(gps):
 # Convert from gps to lps
 def to_gps(local):
 	gps = Point()
-	
+
 	gps.y = math.degrees((local.y + origin_lps.y)/RADIUS)
```

Comment:
Should `utilities.points` have gps transformation utilities since it has other transformations?

Reply:
Don't think so. The `points` module is for generic 2D geometry type stuff. Since the gps transformation requires the origin, I think it is still best implemented as a service.